test/factories/users.rb

```diff
@@ -0,0 +1,80 @@
+require "factory_bot_rails"
+require "faker"
+
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::TvShows::SiliconValley.character.split.first }
+    last_name { Faker::TvShows::SiliconValley.character.split.last }
+    password { "supertopsecretpassword" }
+
+    trait :with_address do
+      address_1 { Faker::Address.street_address }
+      city { Faker::Address.city }
+      region { Faker::Address.state }
+      postal_code { Faker::Address.zip_code }
+      country { "United States" }
+      us_resident { true }
+    end
+
+    trait :with_social_media do
+      github_username { Faker::Internet.username(specifier: 8) }
+      linkedin_username { Faker::Internet.username(specifier: 8) }
+      twitter_username { Faker::Twitter.screen_name }
+    end
+  end
+
+  factory :admin, parent: :user do
+    organization_id { 1 }
+    email { "admin-#{SecureRandom.hex[0, 6]}@codefund.io" }
```

Comment:
My thought process here was since it was a factory, we should use some randomization in the creation and just create a single admin for the seeds. I am fine with whatever option is decided. Two thoughts though:

- I see it being useful in development, someone might create another admin for debugging something.
- Also it seems odd, Is there only supposed to be a single "master" log in for CodeFund admins?

Reply:
We support `N`number of admins. I think I agree with keeping this in random factory form.